specifyweb/specify/migrations/0002_geo.py

```diff
@@ -115,21 +146,24 @@ def create_default_collection_types(apps, schema_editor):
             },
         ),
         migrations.CreateModel(
-            name='CollectionObjectGroupJoin',
+            name='CollectionObjectGroupJoin', # add as dependant to collection object group
             fields=[
                 ('id', models.AutoField(db_column='collectionobjectgroupjoinid', primary_key=True, serialize=False)),
-                ('isprimary', models.BooleanField(blank=True, db_column='IsPrimary', null=True)),
-                ('issubstrate', models.BooleanField(blank=True, db_column='IsSubstrate', null=True)),
+                ('isprimary', models.BooleanField(blank=True, db_column='IsPrimary', null=True)), # TODO: bunsiness rules to only allow one child
```

Comment:
will we do a separate pr for those bus rules? If yes, can we create a new issue in 9.7

Reply:
I added the rules here.  We need to decide if we want to throw an error when setting isPrimary more than once, or just set the others to false when making one true.  I think the second option is good.
https://github.com/specify/specify7/blob/2a715c4ef2a727d1543625e416b931a1f7f1098b/specifyweb/businessrules/rules/cojo_rules.py#L4-L19

Reply:
yes that's kind of what we do with determination